src/documenters/MarkdownDocumenter.ts

```diff
@@ -531,8 +562,13 @@ export class MarkdownDocumenter {
                 );
 
                 if (throwsBlocks.length > 0) {
-                    const heading: string = 'Exceptions';
-                    output.appendNode(new DocHeading({ configuration: this._tsdocConfiguration, title: heading }));
+                    output.appendNode(
+                        new DocParagraph({ configuration }, [
+                            new DocEmphasisSpan({ configuration, bold: true }, [
+                                new DocPlainText({ configuration, text: 'Exceptions:' })
+                            ])
+                        ])
```

Comment:
Heading seems more semantically correct.

Reply:
I'll update this.